vdev/src/commands/check/component_docs.rs

```diff
@@ -0,0 +1,31 @@
+use anyhow::{Result, Ok};
+use crate::git;
+
+
+/// Check that component documentation is up-to-date
+#[derive(clap::Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let files: Vec<String> = git::get_modified_files()?;
+        let dirty_component_files: Vec<String> = files
+            .iter()
+            .cloned()
```

Comment:
You can avoid the `.cloned` by using `.into_iter` which retains ownership of the items.

Reply:
I implemented the into_iter() function